packages/stryker/src/ReporterOrchestrator.ts

```diff
@@ -15,6 +16,7 @@ function registerDefaultReporters() {
   ReporterFactory.instance().register('dots', DotsReporter);
   ReporterFactory.instance().register('clear-text', ClearTextReporter);
   ReporterFactory.instance().register('event-recorder', EventRecorderReporter);
+  ReporterFactory.instance().register('score', ScoreReporter);
```

Comment:
Maybe we can think of a better name? `ScoreReporter` might be confusing as it doesn't explain where it is reporting to. Also: we might be reporting more than just mutation score in the future.

Maybe we can use `BadgeReporter`? Although that has the same problem in the future when we expand the reporting beyond badges. @simondel what is your take in this?

Reply:
Renamed to BadgeReporter